job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -72,6 +72,24 @@ def final REPO_CONFIGS = [
                         "maven.test.failure.ignore": "true"
                 ]
         ],
+        "optaweb-employee-rostering"  : [
+                mvnGoals: "-e -nsu -fae -B -Pwildfly clean deploy com.github.spotbugs:spotbugs-maven-plugin:spotbugs",
+                mvnProps: [
+                        "full"                  : "true",
+                        "container"             : "wildfly",
+                        "integration-tests"     : "true",
+                        "maven.test.failure.ignore" : "true"
+                ]
+        ],
+        "optaweb-vehicle-routing"  : [
+                mvnGoals: "-e -nsu -fae -B -Pwildfly clean deploy com.github.spotbugs:spotbugs-maven-plugin:spotbugs",
+                mvnProps: [
+                        "full"                  : "true",
+                        "container"             : "wildfly",
```

Comment:
`-Dcontainer=wildfly` not needed for Optawebs.

Reply:
DONE